module4/ex10_wedding_planner/ex10_wedding_planner.py

```diff
@@ -1,10 +1,75 @@
-class GuestList:
+from collections import namedtuple
+
+Person = namedtuple('Person', ['first', 'last'])
+
+
+class TableFull(Exception):
+    """Raised when """
     pass
 
 
+class GuestList:
+    max_at_table = 10
+
+    def __init__(self):
+        self.persons = {}
+        self.tables = {}
+
+    def __len__(self):
+        return len(self.persons.keys())
```

Comment:
It is enough to just write `return len(self.persons)` because the length of a dictionary is equal to the number of key-value pairs it has.